documentation/faq.mdx

```diff
@@ -69,29 +69,57 @@ No, these games run on completely different engines and were not made by Naughty
+<!-- https://discord.com/channels/756287461377703987/801612566874554399/1221207684074963014 -->
 
 Most importantly, we are motivated on working on Jak, not other games. Those other games likely already have small niche communities plugging away at things in the background just like we were.
 
 ### Is there support for raytracing?
 
-Implementing raytracing would require a very massive art overhaul which is far outside the scope of this project.
+No. Implementing raytracing would require a very massive art overhaul which is far outside the scope of this project.
 
 ## Game Info
 
+### What versions are supported?
+
+All of them! For both _Jak and Daxter: The Precursor Legacy_ and _Jak II_.
+
 ### Does the game have differences? I don't remember these options!
 
 We have added a plethora of options to the game settings (and removed some that didn't make sense) so that you can have a more up to date experience, or a more PS2-like experience if you decide. It is up to you! There are also a bunch of extra goodies and added secrets to find out. We are aiming to keep the core gameplay (controls, physics, behaviors, etc.) identical however, so if you find any issues or differences with this then do not hesitate to tell us about it.
 
 ### Will there be bug fixes to the game?
 
-The original game had a few issues of varying severity - from progress softlocks to crashing - that we are aiming to patch. The complexity of these issues is varying (if they were simple to reproduce and fix, Naughty Dog likely would have done it), but so far we have fixed more than a couple of issues that could crash the game.
+The original game had a few issues of varying severity - from progress softlocks to crashing - that we have patched or are aiming to patch. The complexity of these issues is varying (if they were simple to reproduce and fix, Naughty Dog likely would have done it), but so far we have fixed more than a couple of issues that could crash the game.
 
 ### Will there be mod support?
 
-In the future yes! We intend to make it as easy as we can to make your own code or import graphics for the game.
+In the future yes! We intend to make it as easy as we can to make your own code or import graphics for the game. In the meanwhile several mods can already be tried out using the unofficial mod launcher, found at 
+
+### Will you support pressure sensitivity?
+
+No. Pressure sensitivity is the possibility of a few controllers such as DualShock2, DualShock3 to give analog states as opposed to not binary states (pressed or not pressed) for face buttons. Even though this seems appealing for driving vehicles, there is no plan to support this, since it requires a massive game rewrite. On top of that, the vast majority of controllers doesn't support this at a hardware or driver level anyway.
```

Comment:
DualShock 2, space.

Also, this isn't limited to face buttons on the DS2. The d-pad, triggers and bumpers all have analog precision, not just the face buttons. It doesn't require a massive game rewrite, so that part can be removed. The fact that only a couple controllers have this feature and how little of it is used in the games is enough of a stated reason.

Adaptive triggers is a very different feature.

Reply:
> how little of it is used in the games

You mean non-Jak games, right?

I am following on the concepts of pressure sensitivity. I found a [good explanation on pressure sensitivity](https://www.pcgamingwiki.com/wiki/Category:Controllers_with_pressure-sensitive_face_buttons) that confirms I understood correctly (phew!); it wasn't easy to find imo, I mainly found _games_:
> Unlike traditional buttons that are simply “on” or “off”, pressure-sensitive face buttons can detect differences in the amount of force applied to them. This means that they can measure the intensity or degree of pressure exerted on the buttons.

Now, where I missed the boat is with adaptive triggers. I thought from the video I shared in the [Discord thread](https://discord.com/channels/756287461377703987/1228112614077235200/1229171507679199283) that they were thresholds of the "rear buttons" (R2, L2) that one could set, also requiring pressure sensitivity. I am wrong, per [The Verge](https://www.theverge.com/2021/3/31/22360812/ps5-adaptive-triggers-dualsense-button-next-gen-immersion).
> But the DualSense changes the game (literally) nonetheless, thanks to the near-magic of its adaptive trigger system, which can adjust the tension of the rear buttons to make it easier (or harder) to press them down in response to gameplay.

That's also why I thought it was _a gigantic lift in terms of the code modification_, because without precise knowledge on the concepts, I assumed Vaser was talking of adaptive triggers.
> it wouldn't be a gigantic lift in terms of the code modification to:
>
> - use the trigger value instead of it being binary to set the pressure sensitivity
> - swap buttons when for example, in a vehicle (R2 for accelerate instead of X)

So, instead I'll just write that in the future a solution as provided by Vaser could be provided to allow for different input styles when driving vehicles, and I'll see in the next review if it's OK :-)

Reply:
"pressure sensitive buttons" in this context just means the buttons track how far down they're pressed, e.g. they are analog instead of binary (but they also have a binary state, but that's irrelevant here).

Reply:
And no, I meant the Jak games. They're used in Jak 1 for three things:
- extra jump height from holding X (including zoomer)
- zoomer turn speed
- debug camera controls

They're used in Jak 2 for:
- extra jump height from holding X
- zoomer gas and zoomer brake
- debug camera controls

Reply:
Okay, it appeared I wasn't following with Vaser when he commented "And the result is playing the game in a completely foreign way to the original." I thought Vaser meant that supporting pressure sensitivity would make the gameplay feel foreign. But I assume now he talking about "tapping the button" that would be foreign?

I'm asking, because I suppose the text written now isn't completely correct and should be reformulated (or slightly). I wrote "Even though this seems appealing for driving vehicles" in the assumption that it could be supported but isn't in the original game. I could write instead "even though it is present in the PS2 and PS3 versions" possibly followed by "for extra jump height and controlling the zoomer" for clarity.

Also, you clarified
> "pressure sensitive buttons" in this context just means the buttons track how far down they're pressed, e.g. they are analog instead of binary (but they also have a binary state, but that's irrelevant here).

I assume this is explained well now (last commit).
> Pressure sensitivity is the possibility of a few controllers such as DualShock 2, DualShock 3 to detect differences in the amount of force applied to them, as opposed to binary inputs (e.g. "pressed" or "not pressed")."

Though I see I had a confusing "not" in the previous iteration.
> Pressure sensitivity is the possibility of a few controllers such as DualShock2, DualShock3 to give analog states as opposed to ~~not~~ binary states (pressed or not pressed) for face buttons.

Reply:
> Okay, it appeared I wasn't following with Vaser when he commented "And the result is playing the game in a completely foreign way to the original." I thought Vaser meant that supporting pressure sensitivity would make the gameplay feel foreign. But I assume now he talking about "tapping the button" that would be foreign?

No I'm saying that the suggestion from users that we should swap `X` with triggers when driving, is a completely foreign way of playing the game.

Reply:
Okay, seems like I've performed worse at interpreting it than I thought -- slightly autistic :p I think it's clear now reading through things again, since the commit will be accepted, I suppose it's not too important anymore. Thanks either way for the clarification!